### ISSUE TITLE
Train and validate the model

### ISSUE THREAD

**Issue by alanjameschapman** [alanjameschapman/pet-image-classifier#6]
As a **developer** I can **train and validate a model** so that I can **prepare it for evaluation using test set**.

**Acceptance Criteria**
- [x] Model performs 'well' on validation set - **otherwise, return to #4** 

**Tasks**
- [x] Augment and engineer features.
- [x] Extract relevant features on which to train the model.
- [x] Train the model on the training set.
- [x] Optimize model hyperparameters.
- [x] Evaluate and indicate the learning curve (loss and accuracy) of the model (LO 5.2) for both training and validation sets.

**Comment by alanjameschapman** [comment]
Task moved from issue #4:
- Extract relevant features on which to train the model.
- Augment and engineer features.

**Comment by alanjameschapman** [comment]
The model has been trained and validated with an accuracy of 80% and acceptable convergence:

![Image](https://github.com/alanjameschapman/pet-image-classifier/assets/137620143/fe1f3c76-2dab-446e-b778-0d7db8f2f8e4)
